pkgs/development/python-modules/pypandoc/default.nix

```diff
@@ -30,6 +32,10 @@ buildPythonPackage rec {
     ./skip-tests.patch
   ];
 
+  nativeBuildInputs = [
+    poetry-core
```

Comment:
Given that we don't use poetry during the build, it seems silly to depend on it.

Reply:
The `pyproject.toml` build uses poetry-core instead of setuptools ([defined on these lines](https://github.com/JessicaTegner/pypandoc/blob/v1.10/pyproject.toml#L39)). I chose to switch to it because the old setuptools build has some unnecessary dependencies (such as pip) in its setup_requires section. I'm not too sure why there are still two different ways to build the package.